easybuild/easyconfigs/s/SignalP/SignalP-4.1g.eb

```diff
@@ -0,0 +1,65 @@
+# Original author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'SignalP'
+version = '4.1g'
+
+homepage = 'https://services.healthtech.dtu.dk/software.php'
+description = """SignalP predicts the presence and location of signal peptide
+    cleavage sites in amino acid sequences from different organisms"""
+
+toolchain = SYSTEM
+
+# No public download URL.  Follow link on above homepage under Protein Sorting >
+# SignalP > Version x.xx > Linux.  This will give a form to complete; after
+# submitting it, you will receive a download page link by e-mail.  The download
+# page will contain two links, one for a .txt licence file and one for a .tar.gz
+# installation file; you should download the latter and move it to the EasyBuild
+# sources directory hierarchy.
+sources = ['%(namelower)s-%(version)s.Linux.tar.gz']
+checksums = ['d72d9fcf3c74e92267bb570b02f4a32a0286826eaf165eed046ad70796db9dc1']
+
+dependencies = [('Perl', '5.36.0', '-minimal', ('GCCcore', '12.2.0'))]
+
+# Change 'scratch' below to your preferred temporary directory path under '/'.
+# We increase the limit on the number of sequences per run from 20k to 250k;
+# increase it further if that will be insufficient for you.
+local_config_cmds = 'mkdir -p %(installdir)s/man1 && mv %(installdir)s/%(namelower)s.1 %(installdir)s/man1 && '
+local_config_cmds += 'sed -i -e "s|^\\(    \\$ENV{SIGNALP} = \'\\).*\\(\';\\)$|\\1%(installdir)s\\2|" '
+local_config_cmds += '-e "s|^\\(my \\$outputDir = \\"/\\).*\\(\\";\\)$|\\1scratch\\2|" '
+local_config_cmds += '-e "s|^\\(my \\$MAX_ALLOWED_ENTRIES=\\).*\\(000;\\)$|\\1250\\2|" %(installdir)s/%(namelower)s'
```

Comment:
please use raw strings here, it'll be a lot more readable

Reply:
@jfgrimm I agree it's not very readable, but sed does require a lot of character escapes to do this, and almost all of these need to be doubled up for being run via eb.  I'm not too clear what you're asking for; is it to avoid the pattern "captures" and thus have something like this example for line 31?  

```local_config_cmds += 'sed -i -e "s|^    \\$ENV{SIGNALP} = \'.*\';$|    \\$ENV{SIGNALP} = \'%(installdir)s\';|" '```